doc/freeband.xml

```diff
@@ -140,3 +140,41 @@ true]]></Example>
     </Description>
   </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="ContentOfFreeBandElement" >
+<ManSection>
+    <Oper Name = "ContentOfFreeBandElement" Arg = "x" /> 
+    <Oper Name = "ContentOfFreeBandElementCollection" Arg = "list" /> 
```

Comment:
`list` is misleading since this could be a semigroup or Green's class or any other collection. Please change this to `coll`